lib/PEX.ts

```diff
@@ -168,8 +170,9 @@ export class PEX {
    *
    * @return the validation results to reveal what is acceptable/unacceptable about the passed object to be considered a valid presentation definition
    */
-  public validateDefinition(presentationDefinition: PresentationDefinitionV1 | PresentationDefinitionV2): Validated {
-    const result = this.definitionVersionDiscovery(presentationDefinition);
+  public validateDefinition(presentationDefinition: IPresentationDefinition): Validated {
+    const presentationDefinitionCopy: IPresentationDefinition = JSON.parse(JSON.stringify(presentationDefinition));
```

Comment:
Not needed here anymore, since definitionDiscovery does the deep copy already

Reply:
fixed